crates/bevy_ecs/src/entity/map_entities.rs

```diff
@@ -51,6 +51,25 @@ pub trait MapEntities {
 /// (mapper inputs) to the current world's entities (mapper outputs).
 ///
 /// More generally, this can be used to map [`Entity`] references between any two [`Worlds`](World).
+///
+/// ## Example
+///
+/// ```
+/// # use bevy_ecs::entity::{Entity, EntityMapper};
+/// # use bevy_utils::EntityHashMap;
+///
```

Comment:
I think you need to add `#` here, otherwise the example will start with an empty line.
```suggestion
/// #
```